src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java

```diff
@@ -186,25 +178,24 @@ private void doSeekTo(RoutingContext routingContext, JsonObject bodyAsJson, Http
                 .map(json -> new TopicPartition(json.getString("topic"), json.getInteger("partition")))
                 .collect(Collectors.toSet());
 
-        Handler<AsyncResult<Void>> seekHandler = done -> {
-            if (done.succeeded()) {
+        BiConsumer<Void, Throwable> seekHandler = (v, ex) -> {
```

Comment:
Wouldn't it be more natural to just use a lambda on line 198 in the call to `whenComplete`, rather than declaring a `BiConsumer` variable here and passing that to `whenComplete`?

Reply:
Totally fine with my, I changed in another place as well and removed the `BiConsumer` import.